packages/world/src/ObjectTypeLib.sol

```diff
@@ -306,7 +306,7 @@ library ObjectTypeLib {
         sig == ITransferSystem.transfer.selector ||
         sig == ITransferSystem.transferTool.selector ||
         sig == ITransferSystem.transferTools.selector ||
-        sig == IMachineSystem.powerMachine.selector;
+        sig == IMachineSystem.fuelMachine.selector;
```

Comment:
does this mean "fuel source"?

Reply:
from my understanding all entities that have their own fuel tank are called "machines" (forcefields, drones, etc)

Reply:
not all smart entities can be refueled though, so I believe we still need a different term for those

Reply:
yee smart entities is more general, since chests and doors are also smart entities like force fields that have programs

machines = smart entities that can have energy
fuel = consumed by machines to increase their energy
food = consumed by players to increase their energy